doc/introduction/mid_circuit_measurements.rst

```diff
@@ -0,0 +1,452 @@
+.. role:: html(raw)
+   :format: html
+
+.. _mid_circuit_measurements:
+
+Mid-circuit measurements
+========================
+
+.. currentmodule:: pennylane.measure
+
+PennyLane allows using measurements in the middle of a quantum circuit.
+Such measurements are called mid-circuit measurements and can be used to
+shape the structure of the circuit dynamically, and to gather information
+about the quantum state during the circuit execution.
+The function to perform a mid-circuit measurement in PennyLane is
+:func:`~.pennylane.measure`, and can be used as follows:
+
+.. code-block:: python
+
+    dev = qml.device("default.qubit")
+
+    @qml.qnode(dev)
+    def my_qnode(x, y):
+        qml.RY(x, wires=0)
+        qml.CNOT(wires=[0, 1])
+        m_0 = qml.measure(1, reset=False, postselect=None)
+
+        qml.cond(m_0, qml.RY)(y, wires=0)
+        return qml.probs(wires=[0]), qml.expval(m_0)
+
+See the following sections for details on
+:func:`~.pennylane.measure`, :func:`~.pennylane.cond`, and statistics
+of mid-circuit measurements, as well as information about simulation
+strategies and how to configure them.
+Additional information can be found in the documentation of the individual
+methods. Also consider our
+:doc:`Introduction to mid-circuit measurements <demos/tutorial_mcm_introduction>`,
+:doc:`how-to on collecting statistics of mid-circuit measurements
+<demos/tutorial_how_to_collect_mcm_stats>`,
+and :doc:`how-to on creating dynamic circuits with mid-circuit measurements
+<demos/tutorial_how_to_create_dynamic_mcm_circuits>`.
```

Comment:
These links did not work :thinking:

Reply:
Similar for the cases further down.

Reply:
I think this was an intermediate broken version, will follow up.